pkgs/by-name/tq/tqftpserv/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  qrtr,
+  meson,
+  zstd,
+  pkg-config,
+  systemd,
+  ninja,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "tqftpserv";
+  version = "1.1";
+
+  buildInputs = [
+    qrtr
+    zstd
+    systemd
+  ];
+
+  src = fetchFromGitHub {
```

Comment:
Can you keep the ordering like in the other packages to have src before the build inputs?